Week3/morejava.js

```diff
@@ -0,0 +1,181 @@
+//Create a function that takes 3 arguments and returns the sum of the these arguments.
+function sum(a, b, c) {
+    return a + b + c;
+}
+console.log(sum(1, 2, 3));
+
+//Create a function named colorCar that receives a color, and prints out, 'a red car' for example.
+function colorCar(x) {
+    return 'a ' + x + ' car';
+}
+console.log(colorCar('red'))
+
+//Create an object and a function that takes the object as a parameter and prints out all of its properties and values.
+let funObject = {
```

Comment:
I don't see you re-assigning this variable anywhere else so we can also use `const` instead of `let` to create this variable

Reply:
👍